frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx

```diff
@@ -605,7 +606,7 @@ describe("ModelDetailPage", () => {
           const action = createMockQueryAction({ model_id: model.id() });
           await setupActions({ model, actions: [action] });
 
-          userEvent.click(screen.getByTestId("new-action-menu"));
+          userEvent.click(screen.getByTestId("actions-menu"));
```

Comment:
If we add `aria-label` we could. I think we should use `test-id` as a last resort. What do you think?
```suggestion
          userEvent.click(screen.getByRole("button", { name: "Actions menu" }));
```